vowpalwabbit/array_parameters.h

```diff
@@ -0,0 +1,172 @@
+#pragma once
+#include <stdio.h>
+#include <iostream>
+#include <stdlib.h>
+#include <string.h>
+
+#ifndef _WIN32
+#include <sys/mman.h>
+#endif
+
+using namespace std;
```

Comment:
I hadn't thought about this, but it makes sense due to uncontrolled namespace pollution.  Other parts of the VW codebase were committing this sin, so I fixed it.  It's a drag though needing to open std in many .cc files.